includes/widgets/common.php

```diff
@@ -364,6 +364,205 @@ protected function register_controls() {
 
 		$this->end_controls_section();
 
+		$this->start_controls_section(
+			'_section_masking',
+			[
+				'label' => __( 'Mask', 'elementor' ),
+				'tab' => Controls_Manager::TAB_ADVANCED,
+			]
+		);
+
+		$this->add_control(
+			'_mask_image',
+			[
+				'label' => __( 'Image', 'elementor' ),
+				'type' => Controls_Manager::MEDIA,
+				'media_type' => 'image',
+				'should_include_svg_inline_option' => true,
+				'library' => 'image/svg+xml',
```

Comment:
Maybe `mime_type`?

Reply:
I tried using the same naming of `wp.media`, and I also thought it might be useful for future use to be able to pass `library` to the media control. ( e.g. when we add our built-in SVG library )

Reply:
Maybe I can handle the `svg` to `image/svg+xml` conversion in the JS, so the PHP will look cleaner.